command/import.go

```diff
@@ -156,36 +155,6 @@ func (c *ImportCommand) Run(args []string) int {
 		return 1
 	}
 
```

Comment:
Part of me wants to keep the `-provider` argument definition above and make it generate a helpful error message below if it's non-empty. :thinking: 

With that said, I don't have a ready-to-go suggestion here for an actionable direction that error message could give. "Specify the provider in the configuration instead" is the gist of it, but I expect that most folks won't really know what to do about it.

So with all of that said, I think it'll be okay to just break this here and then we can describe the migration path in the changelog and/or upgrade guide where we'll have more "space" to describe what the resolution is and link to other relevant documentation.